repos/system_upgrade/common/libraries/tests/test_rhsm.py

```diff
@@ -1,29 +1,119 @@
 from collections import namedtuple
+import os
 
 import pytest
 
 from leapp import reporting
 from leapp.exceptions import StopActorExecutionError
 from leapp.libraries.common import repofileutils, rhsm
-from leapp.libraries.common.testutils import create_report_mocked, logger_mocked
+from leapp.libraries.common.testutils import (
+    create_report_mocked,
+    CurrentActorMocked,
+    logger_mocked
+)
 from leapp.libraries.stdlib import CalledProcessError, api
 from leapp.models import RepositoryFile, RepositoryData
 
 Repository = namedtuple('Repository', ['repoid', 'file'])
 LIST_SEPARATOR = '\n    - '
 
+# External commands called by the RHSM library
+RHSM_LIST_CONSUMED = ('subscription-manager', 'list', '--consumed')
+RHSM_STATUS = ('subscription-manager', 'status')
+RHSM_RELEASE = ('subscription-manager', 'release')
+RHSM_LIST_ENABLED_REPOS = ('subscription-manager', 'repos', '--list-enabled')
+
+RHSM_STATUS_OUTPUT_NONSATELLITE = '''
++-------------------------------------------+
+   System Status Details
++-------------------------------------------+
+Overall Status: Current
+
+System Purpose Status: Matched
+'''
+
+RHSM_STATUS_OUTPUT_SATELLITE = '''
++-------------------------------------------+
+   System Status Details
++-------------------------------------------+
+Overall Status: Current
+
+System Purpose Status: Not Specified
+'''
```

Comment:
I can have the vary same output on non-satellite subscriptions:
```
[root@localhost ~]# rhsm status
+-------------------------------------------+
   System Status Details
+-------------------------------------------+
Overall Status: Current

System Purpose Status: Not Specified
```

Difference between `RHSM_STATUS_OUTPUT_(NON)SATELLITE` outputs is just in the purpose of the system, which is set via the `syspurpose` command (see the [configuring system purpose](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_an_advanced_rhel_installation/configuring-system-purpose-advanced_installing-rhel-as-an-experienced-user) article). What about this?
 - RHSM_STATUS_OUTPUT_NOSCA_NOPURPOSE
 - RHSM_STATUS_OUTPUT_NOSCA_PURPOSE

Or something shorter. I would just remove the satellite/non-satellite at all as I am able to get both outputs with & without satellite.

Reply:
Looking at it now, I absolutely agree that it is absolutely unnecessary to have satellite/non-satellite outputs. Removed them altogether and kept only `RHSM_STATUS_OUTPUT_SCA` and `RHSM_STATUS_OUTPUT_NOSCA`.